crypto/chacha/asm/chacha-armv4.pl

```diff
@@ -34,9 +34,12 @@
 #	but then Snapdragon S4 and Cortex-A8 results get
 #	20-25% worse;
 
+# The first two arguments should always be the flavour and output file path.
+if ($#ARGV < 1) { die "Not enough arguments provided.
+  Two arguments are necessary: the flavour and the output file path."; }
+
 $flavour = shift;
-if ($flavour=~/\w[\w\-]*\.\w+$/) { $output=$flavour; undef $flavour; }
-else { while (($output=shift) && ($output!~/\w[\w\-]*\.\w+$/)) {} }
+$output = shift;
```

Comment:
This PR fix target is the old compiler and only `x86_64`. Can the fix be more scoped to avoid the change on arm file?

Reply:
I explained the reason for changes in all Perl-asm files in a comment above.